hieradata/cluster/chango/role/rke.yaml

```diff
@@ -0,0 +1,6 @@
+---
+classes:
+  - "profile::core::sysctl::rp_filter"
+profile::core::sysctl::rp_filter::enable: false
+profile::core::rke::enable_dhcp: true
```

Comment:
Everything above this line isn't needed on chango.

Reply:
Cleaned up and updated.